lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -288,13 +288,15 @@ public function bindParam($column, &$variable, $type = ParameterType::STRING, $l
 
         $this->boundValues[$column] =& $variable;
 
-        return oci_bind_by_name(
+        if (! oci_bind_by_name(
```

Comment:
This might be better as separate variable assignment than multiline if.

Reply:
I wouldn’t like to introduce a variable just because the function call spans multiple lines.

Let’s leave it like this unless you’re really dislike it.

Reply:
I wouldn’t like to introduce a variable just because the function call spans multiple lines.

Let’s leave it like this unless you’re really dislike it.

Reply:
Looks acceptable: an added single-use variable is more confusing to the reader, IMO